src/harness/testcases/WINNF_FT_S_IPR_testcase.py

```diff
@@ -1399,10 +1400,12 @@ def test_WINNF_FT_S_IPR_8(self, config_filename):
           })
     using_esc_dpa = 'escDpa' in config
     using_portal_dpa = 'portalDpa' in config
+    using_gb_dpa = 'gbDpa' in config
     self.assertTrue(
-        using_esc_dpa ^ using_portal_dpa,
+        using_esc_dpa ^ using_portal_dpa ^ using_gb_dpa,
```

Comment:
this condition should be
(a ^ b ^ c) and not (a and b and c)
since we want one and only one to be true

Reply:
Changed to something simpler (assertequal(1, a + b + c))